Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.Source.cs

```diff
@@ -106,6 +106,14 @@ private static bool IsHttpUri(Uri uri)
                 if (IsCacheEnabled && _isHttpSource)
                 {
                     _image.Source = await ImageCache.GetFromCacheAsync(_uri);
+                    if (_image.Source != null)
```

Comment:
That would still be a breaking change just smaller :) Anyhow, I think there is no way to do this without breaking change. Can we make custom events and mark existing as obsolete?

Also setting the source to already loaded BitmapImage from code behind does not signal the events:
`var bmp = new BitmapImage();
            bmp.SetSource(stream);
            TestImageEx.Source = bmp;`
But this is probably OK since it is the same behavior as on build-in Image control.